src/main/scala/no/ndla/articleapi/controller/InternController.scala

```diff
@@ -60,5 +64,41 @@ trait InternController {
     get("/embedurls/:external_subject_id") {
       ArticleContentInformation.getExternalEmbedResources(params("external_subject_id"))
     }
+
+    get("/reports/headerElementsInLists") {
+      contentType = "text/csv"
+      logger.info("Start searching for header elements in Lists in all articles")
+      val start = System.currentTimeMillis()
+      var errorMessages: List[HtmlFaultRapport] = immutable.List()
+      articleRepository.getAllIds.map(m => {
+        val article = readService.withId(m.articleId)
+          article match {
+          case Some(art) => {
+            art.content.map(c => {
+
+              val listElements = stringToJsoupDocument(c.content).select("li")
+              val regex = """<h[1-4]>""".r
+
+              listElements.map(li => {
+                val allIn = regex.findAllIn(li.toString).toList
```

Comment:
Det tenkte jeg ikke på engang at jeg kan lete etter h taggene med JSoup istedet for regexp. Kan sjekke opp.